tsl/src/continuous_aggs/create.h

```diff
@@ -12,6 +12,7 @@
 #include "ts_catalog/continuous_agg.h"
 
 #define CONTINUOUS_AGG_CHUNK_ID_COL_NAME "chunk_id"
+#define CAGG_MAX_JOIN_RELATIONS 2
```

Comment:
```suggestion
#define CONTINUOUS_AGG_MAX_JOIN_RELATIONS 2
```
Just to follow the other pattern!

Reply:
Another comment, do we need to put it into the header??? I mean adding this to the header will be available for other modules that include this and IMHO this is restricted to create.c

Reply:
IMHO as we extend this feature, we might need this in invalidations, materialize, etc. so header is a better place.